lib/tasks/populate.rake

```diff
@@ -115,6 +115,39 @@ namespace :db do
   Category.first.docs.first.update(title: "Näide toetust dokumentatsiooni tõlgitud prantsuse" ,body: '', locale: :et)
   Category.first.docs.first.update(title: "Documentació de suport Exemple traduïda al francès" ,body: '', locale: :ca)
 
+  # Create document comment threads for our users
+  Doc.all.each do |doc|
+
+    f = Forum.where(name: 'Doc comments').first
+    rand(0..2).times do
+      topic = f.topics.create!(
+        name: build_question(Faker::Hacker.ingverb + " " + Faker::Hacker.noun),
+        user_id: User.where(admin: false).sample.id,
+        doc_id: doc.id
+      )
+      post = topic.posts.create!(
+        body: Faker::Lorem.paragraphs(rand(1..2)).join('<br/><br/>'),
+        user_id: topic.user_id,
+        kind: 'first'
+      )
+
+      timeseed = rand(1..30)
+      Timecop.travel(Date.today-timeseed.days)
```

Comment:
Doesn't impact much, but using `timeseed.days.ago` is more "Railsy", clear and robust, because it obeys the server's timezone. Not a problem here I suppose.